fluent-plugin-opensearch.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'rake', '>= 0'
   s.add_development_dependency 'webrick', '~> 1.7.0'
-  s.add_development_dependency 'webmock', '~> 3'
+  s.add_development_dependency 'webmock', '= 3.18.1'
```

Comment:
How about pinning webmock version with `~> 3.18.1`. Precisely version locking is sick for us.